common/djangoapps/student/management/commands/set_superuser.py

```diff
@@ -0,0 +1,47 @@
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+import re
+
+
+class Command(BaseCommand):
+    option_list = BaseCommand.option_list + (
+        make_option('--unset',
+                    action='store_true',
+                    dest='unset',
+                    default=False,
+                    help='Set is_superuser to False instead of True'),
+    )
+
+    args = '<user|email> [user|email ...]>'
+    help = """
+    This command will set is_superuser to true for one or more users.
+    Lookup by username or email address, assumes usernames
+    do not look like email addresses.
+    """
+
+    def handle(self, *args, **options):
+        if len(args) < 1:
+            raise CommandError('Usage is set_superuser {0}'.format(self.args))
+
+        for user in args:
```

Comment:
optional consideration: the way things are set up right now, if you pass ten users and get a failure midway through, you'll need to scrutinize the command + output and figure out what state things were left in.  if atomicity is an issue you could wrap the whole loop in a manual transaction; you might also just want to output some feedback when the `save()` call succeeds so you know who was updated and who wasn't.